test/test_priority_queue.jl

```diff
@@ -176,6 +176,12 @@ end
     @test dequeue_pair!(pq) == ('e'=> 1)
     @test dequeue_pair!(pq, 'b') == ('b'=>4)
     @test length(pq) == 3
+
+    # delete!
+    pq = PriorityQueue(["a","b","c"],[2,3,1],Base.Order.Forward)
```

Comment:
Same change needed here.

Reply:
That's sad to see that https://github.com/JuliaDocs/Documenter.jl/issues/527 is not fixed!
We should have been able to catch this kind of error!

Reply:
Maybe some people at JuliaCon hackaton should try to fix thiss first because that's very annoying not to be able to rely on doctests!